CONTRIBUTING.rst

```diff
@@ -539,19 +543,19 @@ Pull Requests
   
 * At times it may be necessary to temporarily add a commit to a PR branch e.g.,
   to facilitate testing. These commits should be removed prior to landing the
-  PR and their title is prefixed with ``DELETE ME:``.
+  PR and their title is prefixed with ``drop! ``.
```

Comment:
Inconsistent use of spaces after commit prefixes.

Reply:
Will do.

Reply:
Wait, the new version is correct. The prefix must end end in a space. But when I later refer to these types of commits the space can be omitted.